antora-ui-camel/src/css/release.css

```diff
@@ -11,3 +11,9 @@
   grid-column-start: 2;
   margin-left: 1rem;
 }
+
+@media screen and (max-width: 1023px) and (min-width: 480px) {
+  .release dl {
+    overflow: scroll;
+  }
+}
```

Comment:
Why use a media query, could we put this in `.release dl` instead? What would be the downside of that?

Reply:
No, the downside of this. It will work fine without using the media query.